### ISSUE TITLE
Remove redundant aria-describedby attribute from link and summary paragraph type

### ISSUE THREAD

**Issue by danchamp** [localgovdrupal/localgov_base#141]
The aria-describedby attribute has an invalid value (should be an element id) and is redundant:

https://github.com/localgovdrupal/localgov_base/blob/12c21a3a28710c3031333e28ea7ef786202fe161/templates/paragraphs/paragraph--localgov-link-and-summary.html.twig#L69

[Accessibility audit: issue 43](https://docs.google.com/spreadsheets/d/1t8rRIQe_1kogxPBZDKcsbf8YxtEfC9iGJvidkagNSXY/edit#gid=1779042480&range=D44)
